Ollivanders/src/net/pottercraft/Ollivanders2/Potion/IngredientType.java

```diff
@@ -7,35 +7,51 @@
    BEZOAR (Material.COAL, (short)1, "Bezoar"), // charcoal
    BILLYWIG_STING_SLIME (Material.SLIME_BALL, (short)0, "Billywig Sting Slime"),
    BOOM_BERRY_JUICE (Material.POTION, (short)11, "Boom Berry Juice"),
+   BOOMSLANG_SKIN (Material.ROTTEN_FLESH, (short)0, "Boomslang Skin"),
    BONE (Material.BONE, (short)0, "Bone"),
    BLOOD (Material.POTION, (short)7, "Blood"),
    CHIZPURFLE_FANGS (Material.PUMPKIN_SEEDS, (short)0, "Chizpurfle Fangs"),
    DEATHS_HEAD_MOTH_CHRYSALIS (Material.COAL, (short)0, "Death's Head Moth Chrysalis"),
    DEW_DROP (Material.GHAST_TEAR, (short)0, "Dew Drop"),
    DITTANY (Material.SAPLING, (short)2, "Dittany"), //birch sapling
+   DRAGON_BLOOD (Material.SPLASH_POTION, (short)1, "Dragon Blood"),
+   DRAGONFLY_THORAXES (Material.BEETROOT_SEEDS, (short)0, "Dragonfly Thoraxes"),
    DRIED_NETTLES (Material.SAPLING, (short)0, "Dried Nettles"), // oak sapling
+   FAIRY_WING (Material.GOLD_NUGGET, (short)0, "Fairy Wing"),
+   FIRE_SEEDS (Material.NETHER_WARTS, (short)0, "Fire Seeds"),
    FLOBBERWORM_MUCUS (Material.SLIME_BALL, (short)0, "Flobberworm Mucus"),
+   FLUXWEED (Material.DEAD_BUSH, (short)0, "Fluxweed"),
+   FULGURITE (Material.GLOWSTONE_DUST, (short)0, "Fulgurite)"),
    GALANTHUS_NIVALIS (Material.RED_ROSE, (short)3, "Galanthus Nivalis"), //azure bluet
    GINGER_ROOT (Material.BEETROOT, (short)0, "Ginger Root"),
+   GROUND_DRAGON_HORN (Material.GLOWSTONE_DUST, (short)0, "Ground Dragon Horn"),
    GROUND_PORCUPINE_QUILLS (Material.INK_SACK, (short)3, "Ground Porcupine Quills"),
    GROUND_SCARAB_BEETLE (Material.SULPHUR, (short)0, "Ground Scarab Beetle"),
    GROUND_SNAKE_FANGS (Material.INK_SACK, (short)15, "Ground Snake Fangs"),
    HONEYWATER (Material.POTION, (short)0, "Honeywater"),
    HORKLUMP_JUICE (Material.DRAGONS_BREATH, (short)0, "Horklump Juice"),
    HORNED_SLUG_MUCUS (Material.SLIME_BALL, (short)0, "Horned Slug Mucus"),
+   HORN_OF_BICORN (Material.BLAZE_ROD, (short)0, "Horn of Bicorn"),
    INFUSION_OF_WORMWOOD (Material.POTION, (short)5, "Infusion of Wormwood"),
    JOBBERKNOLL_FEATHER (Material.FEATHER, (short)0, "Jobberknoll Feather"),
+   Knotgrass (Material.LONG_GRASS, (short)0, "Knotgrass"),
+   LACEWING_FLIES (Material.PUMPKIN_SEEDS, (short)0, "Lacewing Flies"),
    LAVENDER_SPRIG (Material.DOUBLE_PLANT, (short)1, "Lavender Sprig"), //lilac
+   LEECHES (Material.INK_SACK, (short)0, "Leeches"),
    LETHE_RIVER_WATER (Material.POTION, (short)0, "Lethe River Water"), //bottle of water
    LIONFISH_SPINES (Material.RAW_FISH, (short)0, "Lionfish Spines"),
    MANDRAKE_LEAF (Material.WATER_LILY, (short)0, "Mandrake Leaf"),
+   MERCURY (Material.INK_SACK, (short)0, "Mercury"),
    MINT_SPRIG (Material.MELON_STEM, (short)0, "Mint Sprig"),
    MISTLETOE_BERRIES (Material.INK_SACK, (short)3, "Mistletoe Berries"),
    MOONDEW_DROP (Material.GHAST_TEAR, (short)0, "Moondew Drop"),
+   PHILOSOPHERS_STONE (Material.REDSTONE, (short)0, "Philosophers Stone"),
```

Comment:
I am not sure I would make this an ingredient right now because then it would be consumed in the making of the potion - which it should not be if they can keep using it to make more Elixir in the book. 

I expect for Elixir of life we will need to get in to more advanced brewing that can do things like require you add the stone but it is not used up in the potion brewing process.

Reply:
Yeah, I'll add that to the "classic error" list, new term I just learned from Autumn :D 
Advanced brewing does sound like a fun thing to build, not sure how much I'd be able to help with it at this juncture though

Reply:
Removed.